app/script/media/MediaEmbeds.js

```diff
@@ -100,7 +100,7 @@ z.media.MediaEmbeds = (function() {
         .substr(url.indexOf('?'), url.length)
         .replace(/^[?]/, '&')
         .replace(/[&]v=[a-zA-Z0-9_-]{11}/, '')
-        .replace(/[&#]t=([a-z0-9]+)/, (a, b) => `&start=${_convert_youtube_timestamp_to_seconds(b)}`)
+        .replace(/[&#]t=([a-z0-9]+)/, (first, second) => `&start=${_convert_youtube_timestamp_to_seconds(second)}`)
```

Comment:
To find a better name for `first` and `second` I need to know what they are (values). :D